pkg/util/runtime/features.go

```diff
@@ -52,6 +52,9 @@ const (
 	// FeaturePlayerTracking is a feature flag to enable/disable player tracking features.
 	FeaturePlayerTracking Feature = "PlayerTracking"
 
+	// FeatureAutopilotPassthroughPort is a feature flag that enables/disables Passthrough Port Policy.
+	FeatureAutopilotPassthroughPort Feature = "PassthroughPortPolicy"
```

Comment:
I would put this in `Dev features`, and follow the instructions on line 73-76. In particular, you're missing the `install/helm/agones` change, which you'll need when we add the webhook to make it conditional in helm.

After the next step (when you add the webhook), we can go ahead and move it to Alpha. See lines 79-83 for those instructions, but it's basically a couple of modifications to ensure we test with it enabled.

Reply:
Ohhh okok i missed all this section of code, sg, thanks!